src/undomanager.js

```diff
@@ -2,11 +2,13 @@ define([
     'jquery',
     'underscore',
     'tpl!vellum/templates/undo_alert',
+    'vellum/util',
 ], function(
     $,
     _,
-    UNDO_ALERT
-) {
+    UNDO_ALERT,
+    util,
+undefined) {
```

Comment:
Vellum convention is to have templates at the end. Also I don't think that you need the `undefined` here.